prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails
+Estimate: 70 minutes
+Actual:   83 minutes
+"""
+
+
+def main():
+    email_to_name = {}
+
+    email = input("Email: ")
+    while email != "":
+        name = convert_email_to_name(email)
+        action = input(f"Is your name {email_to_name.get(email)}? (Y/N) ").upper()
+        if action == "N" or action == "NO":
```

Comment:
Could work but, have you read the wording in the practical? 
"In our program, if the user does not press Enter or Y, then ask for their name"